src/app/shared/entity/node.ts

```diff
@@ -247,6 +247,13 @@ export class VSphereNodeSpec {
   memory: number;
   template: string;
   diskSizeGB?: number;
+  tags?: VSphereTag[];
+}
+
+export class VSphereTag {
+  name: string;
+  description: string;
+  categoryID?: string; // Backend: CategoryID when empty the default category will be used.
```

Comment:
I would add a note that it can be omitted or even omit it completely if it is not used in the UI to avoid confusion.

Reply:
fixed [`e45397a` (#4430)](https://github.com/kubermatic/dashboard/pull/4430/commits/e45397a2d20b2894d59f37e7087c1c21fa620415)